services/frontend-service/pkg/handler/handle_test.go

```diff
@@ -592,18 +600,25 @@ func (m *mockDeployClient) Deploy(_ context.Context, in *api.DeployRequest, _ ..
 	return &emptypb.Empty{}, nil
 }
 
-func (m *mockDeployClient) ReleaseTrain(_ context.Context, in *api.ReleaseTrainRequest, _ ...grpc.CallOption) (*emptypb.Empty, error) {
+func (m *mockDeployClient) ReleaseTrain(_ context.Context, in *api.ReleaseTrainRequest, _ ...grpc.CallOption) (*api.ReleaseTrainResponse, error) {
 	m.releaseTrainRequest = in
-	return &emptypb.Empty{}, nil
+	// TODO: get upstream from repo state
```

Comment:
The comment can stay but remove the TODO tag

Reply:
Removed